adding help numbers

ARMSTRONGOPONDO opened the pull request:
Added Help Page Links and Fixed Display in Summary Dialog

 Linked help pages to the following dialogs and sub-dialog:
    dlgTraits (701)
    dlgClimBoxCorrelations (703)
    dlgImportClimOb (650)
    sdgImport (716)

Corrected display logic in the 2/3-Way Summary dialog (`dlgDescribeTwoVariable`) under the Describe menu:
    Ensured correct alignment of row and column variables in summary tables
    Fixed mapping of receiver variable (`.x`) to match expected R output

Note: This change includes cleanup after help ID mix-up from earlier commit #9551.

lilyclements reviewed (commented):
@ARMSTRONGOPONDO looks good so far - are you doing any other help IDs (is this ready for review/merge?)

rdstern reviewed (commented):
@ARMSTRONGOPONDO 
a)The links to Traits and Correlations work fine.
b) The define should have 672 as on the board.  It is currently going to the wrong define.
c) Import from Climmob was also added - please include it.  The number is 650 and the number for the sub-dialog is 715

ARMSTRONGOPONDO commented:
@lilyclements Added Help Page Links and Fixed Display in Summary Dialog

Linked help pages to the following dialogs and sub-dialog:
dlgTraits (701)
dlgClimBoxCorrelations (703)
dlgImportClimOb (650)
sdgImport (716)
and corrected the define from 328 to 672 you stated it was wrong 

Corrected display logic in the 2/3-Way Summary dialog (dlgDescribeTwoVariable) under the Describe menu:
Ensured correct alignment of row and column variables in summary tables
Fixed mapping of receiver variable (.x) to match expected R output

Note: This change includes cleanup after help ID mix-up from earlier commit #9551.
